### ISSUE TITLE
Problem with Build SFX in 0.16.27+

### ISSUE THREAD

**Issue by davinkevin** [jspm/jspm-cli#1551]
Hi, 

I have a problem since the upgrade in version 0.16.27 with the following error : 
```
Error: TypeError: Global SFX bundle dependency "github:angular/bower-angular@1.4.8.js" must be configured to an environment global via the globalDeps option.
    at /Users/kevin/git/AngularStomp/build/tasks/build.js:25:27
    at tryCatcher (/Users/kevin/git/AngularStomp/node_modules/jspm/node_modules/systemjs-builder/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/kevin/git/AngularStomp/node_modules/jspm/node_modules/systemjs-builder/node_modules/bluebird/js/release/promise.js:497:31)
    at Promise._settlePromise (/Users/kevin/git/AngularStomp/node_modules/jspm/node_modules/systemjs-builder/node_modules/bluebird/js/release/promise.js:555:18)
    at Promise._settlePromise0 (/Users/kevin/git/AngularStomp/node_modules/jspm/node_modules/systemjs-builder/node_modules/bluebird/js/release/promise.js:600:10)
    at Promise._settlePromises (/Users/kevin/git/AngularStomp/node_modules/jspm/node_modules/systemjs-builder/node_modules/bluebird/js/release/promise.js:679:18)
    at Async._drainQueue (/Users/kevin/git/AngularStomp/node_modules/jspm/node_modules/systemjs-builder/node_modules/bluebird/js/release/async.js:125:16)
    at Async._drainQueues (/Users/kevin/git/AngularStomp/node_modules/jspm/node_modules/systemjs-builder/node_modules/bluebird/js/release/async.js:135:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/kevin/git/AngularStomp/node_modules/jspm/node_modules/systemjs-builder/node_modules/bluebird/js/release/async.js:16:14)
    at processImmediate [as _immediateCallback] (timers.js:358:17)
```
=> https://travis-ci.org/davinkevin/AngularStompDK/builds/110818469

All seems to be ok with the globalDeps and was working in 0.16.25, version I use before.

Does something change in the global deps gestion ? 

Thanks for your help

**Comment by DrDanRyan** [comment]
I have the same issue with my bundle of the form:
`jspm bundle-sfx .jspm/entry.js - angular - lodash --globals "\{angular: 'angular', lodash: 'lodash'\}"` --skip-source-maps --no-runtime

**Comment by davinkevin** [comment]
I forgot to add the build task I use (it's always the same :smile: ) : https://github.com/davinkevin/AngularStompDK/blob/master/build/tasks/build.js
```
gulp.task('build:jspm', (cb) => {
    let options = {
        sourceMaps: true,
        globalDeps: { angular : 'angular', stompjs : 'Stomp' },
        formatHint: true
    };

    new jspm.Builder()
        .buildStatic(`${paths.app.entryPoint} - angular - stompjs`, `${paths.releaseDir}/${paths.app.name}.js`, options)
        .then(() => cb())
        .catch((ex) => cb(new Error(ex)));
});
```

**Comment by guybedford** [comment]
Fixed in https://github.com/systemjs/builder/commit/9b1d18cfadf4083cf5c88329c9a9e42a19bd7268 pending release.

**Comment by davinkevin** [comment]
Thanks :+1:

**Comment by guybedford** [comment]
Released in 0.16.30.
